docs/javascript/google-analytics.js

```diff
@@ -1,7 +0,0 @@
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
How come we're removing GA?

Reply:
Swapped it out w/ Marketing's analytics.